packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -140,4 +164,27 @@ export default class QuanticResultQuickview extends LightningElement {
   get buttonLabel() {
     return this.hasNoPreview ? this.labels.noPreview : this.labels.openPreview;
   }
+
+  get buttonClass() {
```

Comment:
I think this can be simplified if you make the possible variant values: `'brand' | 'outline-brand'`;
```suggestion
  get buttonClass() {
    const variantClass = this.previewButtonVariant
      ? `slds-button_${previewButtonVariant}`
      : 'quickview__button-base';
    return `slds-button ${variantClass}`;
  }
```

Reply:
You don't like my suggestion? :P